modules/installation-ibm-cloud-iam-policies-api-key.adoc

```diff
@@ -64,14 +73,87 @@ You must assign the required access policies to your IBM Cloud account.
 3. Only one service is required. The service that is required depends on the type of cluster that you are installing. If you are installing a public cluster, `Internet Services` is required. If you are installing a private cluster, `DNS Services` is required.
 --
 //TODO: IBM confirmed current values in the table above. They hope to provide more guidance on possibly scoping down the permissions (related to resource group actions).
+endif::ibm-vpc[]
+
+ifdef::ibm-power-vs[]
+[id="pre-requisite-permissions-ibm-cloud_{context}"]
+== Pre-requisite permissions
+
+.Pre-requisite permissions
+[cols="1,2",options="header"]
+|===
+|Role |Access
+
+|Viewer, Operator, Editor, Administrator, Reader, Writer, Manager
+|Internet Services service in <resource_group> resource group
+
+|Viewer, Operator, Editor, Administrator, User API key creator, Service ID creator
+|IAM Identity Service service
+
+|Viewer, Operator, Administrator, Editor, Reader, Writer, Manager, Console Administrator
+|VPC Infrastructure Services service in <resource_group> resource group
+
+|Viewer
+|<resource_group> resource group: resourceType string equals resource-group, resource string equals <resource_group>
```

Comment:
@stevsmit request you to provide an example to understand how exactly above part should updated.

Reply:
```suggestion
|<resource_group> resource group: `resourceType` string equals resource group. Resource string equals <resource_group>
```

^ I think this particular description is confusing to understand, but it might be my lack of knowledge on the topic. Would we also but "resource string" in back ticks? eg, should it be "`resourceString`" ?

Reply:
@mjpytlak request your inputs here. Thanks.

Reply:
The above content was provided in this comment : https://github.com/openshift/openshift-docs/pull/47208#issuecomment-1176792430
@clnperez request you to review above content and please let me know if you agree to the above suggestions. Thanks.

Reply:
it is confusing. :D maybe we don't need the title at the beginning of the line (_<resource_group> resource group:_), and use this wording instead:
|Resource group: Access to a Service based on a resource group. The access scope should be `specific resources` with an attribute type of `Resource group` and value equal to <your_resource_group>

Reply:
Updated it as : 
```
|<resource_group> resource group: `resourceType` string equals resource group, `resourceString` equals <your_resource_group_name>
```
cc : @clnperez

Reply:
Updated that stated above. Thanks @clnperez.
cc : @stevsmit